pkg/vm/engine/tae/logstore/store/checkpoint.go

```diff
@@ -151,6 +151,7 @@ func (w *StoreImpl) CkpCkp() {
 	if err != nil {
 		panic(err)
 	}
+	e.Free()
```

Comment:
I don't think here, you are doing remove parse error.    This is a big thing, you cannot slip it through a PR.
This MUST be part of PR commit message.

Reply:
Ok, I'll move it to another pr.

Reply:
Our PR commit message is annoying.   I agree.   But this is important change/fix.   Either update the PR message, or another PR is fine tool

Thank you.